api/services/control/control.proto

```diff
@@ -62,6 +62,7 @@ message SolveRequest {
 	CacheOptions Cache = 8 [(gogoproto.nullable) = false];
 	repeated string Entitlements = 9 [(gogoproto.customtype) = "github.com/moby/buildkit/util/entitlements.Entitlement" ];
 	map<string, pb.Definition> FrontendInputs = 10;
+	Pin Pin = 11;
```

Comment:
I think rather than "pins" it is better to think about this as "source policy".

Reply:
Renamed to "source policy"